readme.md

```diff
@@ -483,13 +484,16 @@
 - [Non-Financial Blockchain](https://github.com/machinomy/awesome-non-financial-blockchain) - Non-financial blockchain applications.
 - [Mastodon](https://github.com/tleb/awesome-mastodon) - Open source decentralized microblogging network.
 
+## Conferences, Meetups
```

Comment:
Both entries would be helped by adding the description. What type of conferences, who is speaking on what?